configuration/monitor-acs.adoc

```diff
@@ -1,14 +1,46 @@
 [id="monitor-acs"]
-= Monitoring with Prometheus
+= Monitoring ACS
 include::modules/common-attributes.adoc[]
 :context: monitor-acs
 
 toc::[]
 
+= Monitoring with OpenShift 4
+
+[role="_abstract"]
+link:https://docs.openshift.com/container-platform/4.13/monitoring/monitoring-overview.html[OpenShift Container Platform] includes a preconfigured,
+preinstalled, and self-updating monitoring stack that provides monitoring for core platform components.
+{product-title-short} has support for OpenShift Monitoring via encrypted and authenticated endpoint accessible only by Prometheus Operator.
+
+[IMPORTANT]
+====
+If you have previously configured monitoring with Prometheus Operator you may consider disabling it as it will be enabled by default on OpenShift 4.
+Having multiple ServiceMonitors could result in duplicating data and wrong query results.
+====
+
+include::modules/enable-openshift-monitoring-central-operator.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_ocp/install-central-config-options-ocp.adoc#install-central-config-options-ocp[Central configuration options using the Operator]
+
+include::modules/enable-openshift-monitoring-central-helm.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_other/install-central-other.adoc#change-config-options-after-deployment-central-services_install-central-other[Changing configuration options after deploying the central-services Helm chart]
+
+= Monitoring with Prometheus
+
 [role="_abstract"]
 link:https://prometheus.io/[Prometheus] is an open-source monitoring and alerting platform.
 You can use it to monitor health and availability of Central and Sensor components of {rh-rhacs-first}. When you enable monitoring, {product-title-short} creates a new monitoring service on port number `9090` and a network policy allowing inbound connections to that port.
 
+[NOTE]
+====
+Monitoring service is unencrypted and has no authorization. It should be used only when you do not want to use OpenShift Monitoring.
```

Comment:
```suggestion
This monitoring service is unencrypted and has no authorization. Use this only when you do not want to use {osp} monitoring.
```

Reply:
Maybe a bit pedantic, but what does the "service is unencrypted" mean? Isn't the service resource always a plaintext manifest? I think it's better to be more precise here and state that the service exposes an endpoint that is not encrypted by TLS.